File: python/problem1117.py
I = 0
nota = 0
while I < 2:
    x = float(input())
    if x < 0 or x > 10:
        print('nota invalida')
    else:
        nota += x
        I += 1

media = nota/2

print('media = %.2f' % media)

File: java/Problem1117.java
import java.util.Scanner;

public class Problem1117 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int I = 0;
        double total = 0;
        while (I < 2) {
            double nota = sc.nextDouble();
            if (nota < 0 || nota > 10) {
                System.out.println("nota invalida");
            } else {
                total += nota;
                I++;
            }
        }

        System.out.printf("media = %.2f\n", total/2.0);

        sc.close();
    }
}